src/hub.js

```diff
@@ -569,6 +574,13 @@ document.addEventListener("DOMContentLoaded", async () => {
   });
 
   const socket = connectToReticulum(isDebug);
+
+  socket.onClose(() => {
+    // The socket should only close of the server has explicitly killed it.
+    entryManager.exitScene();
+    remountUI({ roomUnavailableReason: "left" });
```

Comment:
@gfodor I re-used the existing "You have left the room" message here, dunno if we want different messaging.

Reply:
i think they should know they were kicked because (since they can refresh anyway) the main thing kicking is doing is giving the owner a way to nudge them to behave better. (once we have OAuth in place, it can become a ban)

Reply:
Made it "You have been kicked from the room"